### DISCUSSION TITLE
Changing color of drawn path?

### DISCUSSION THREAD

**Discussion by NoBagels** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#702] [Q&A] [answered]
Hello,

Is it possible to change the color of the robot's path (i.e. where it has moved/cleaned), and if so, how?

The reason for asking is that it's not really visible for some of my rooms, so ideally I'd want a black path drawn..

![bild](https://github.com/PiotrMachowski/lovelace-xiaomi-vacuum-map-card/assets/22916423/1af0d31a-688d-4e96-ac33-b00f58f52529)

Thanks, and sorry if I've missed it in the documentation..

**Comment by PiotrMachowski** [comment, marked as answer]
It's something that has to be changed in the integration that provides the map

**Reply by NoBagels** [reply]
Ah, of course. That solved it, thanks!
